lib/core/services/people-content.service.ts

```diff
@@ -16,13 +16,14 @@
  */
 
 import { Injectable } from '@angular/core';
-import { Observable, from, throwError } from 'rxjs';
+import { Observable, from, throwError, of } from 'rxjs';
 import { AlfrescoApiService } from './alfresco-api.service';
-import { catchError, map } from 'rxjs/operators';
-import { PersonEntry, PeopleApi, PersonBodyCreate, Pagination, PersonBodyUpdate } from '@alfresco/js-api';
+import { catchError, map, tap } from 'rxjs/operators';
+import { PeopleApi, PersonBodyCreate, Pagination, PersonBodyUpdate } from '@alfresco/js-api';
 import { EcmUserModel } from '../models/ecm-user.model';
 import { LogService } from './log.service';
 import { AuthenticationService } from './authentication.service';
+import { ContentService } from './content.service';
 
 // eslint-disable-next-line no-shadow
 export enum ContentGroups {
```

Comment:
can you just move the enum to a separate file? this will allow not using the eslit-disable hack